src/conversions/hashbrown.rs

```diff
@@ -194,8 +198,7 @@ mod tests {
             let mut map = hashbrown::HashMap::<i32, i32>::new();
             map.insert(1, 1);
 
-            let m = map.to_object(py);
-            let py_map = m.downcast_bound::<PyDict>(py).unwrap();
+            let py_map = (&map).into_pyobject(py).unwrap();
```

Comment:
❤️ it's nice to see the downcasts removed, even if there's suddenly a new fallible case!

Reply:
Yes! And getting a `Bound` directly instead of a `Py` is also super nice